src/utils/createMusicMetadata.ts

```diff
@@ -1,13 +1,15 @@
+import { FormData } from "@/components/MintForm"
+
 const music_metadata = {
-	name: '', // music nft
+	name: '', // artist name - title
 	description: '', // music NFT with data on-chain as Base64 encoded string.
-	image: '', // ipfs://bafkreibjsjxxys2pl6wijorije7yixjsw7xlho5krqdimwayyobnybewey
+	image: undefined, // ipfs://bafkreibjsjxxys2pl6wijorije7yixjsw7xlho5krqdimwayyobnybewey
```

Comment:
this MUST be defined as a string to follow the music metadata standards 

https://gist.github.com/bretth18/df8358c840fa94946ec212f753e290dd